src/AWS/DynamoDB/DynamoDb.purs

```diff
@@ -0,0 +1,100 @@
+module AWS.DynamoDb where
+
+import Prelude
+import AWS.Core.Client (makeClientHelper)
+import AWS.Core.Types (DefaultClientProps)
+import Control.Monad.Error.Class (class MonadError)
+import Control.Promise (Promise)
+import Control.Promise as Promise
+import Data.Either (Either(..))
+import Data.Function.Uncurried (Fn2, runFn2)
+import Data.Maybe (Maybe(..))
+import Data.Nullable (Nullable)
+import Data.Nullable as Nullable
+import Effect (Effect)
+import Effect.Aff (Aff, Error, catchError)
+import Effect.Class (liftEffect)
+import Foreign (Foreign)
+import Justifill.Fillable (class Fillable)
+import Justifill.Justifiable (class Justifiable)
+import Type.Proxy (Proxy(..))
+import Justifill (justifillVia)
+
+foreign import data DynamoDbClient :: Type
+
+foreign import newDynamoDbClient :: Foreign -> (Effect DynamoDbClient)
+
+catchAwsError :: forall t45 t49 t51. MonadError t49 t45 => t45 (Either t49 t51) -> t45 (Either t49 t51)
```

Comment:
No other way than having t45 t49 t51 there?

Reply:
I think this can be adjusted. I'll have a look.

Reply:
Adjusted